web/admin/src/Admin.js

```diff
@@ -196,27 +194,21 @@ export default class Admin extends Component {
       })
 
       modRef.on('child_added', data => {
-        this.setState({ moderator: [...this.state.moderator, { ...data.val(), key: data.key }] })
+        this.setState({ moderator: { ...data.val(), key: data.key } })
```

Comment:
I don't think I understand this change.  Each `child_added` will replace the state's `moderator`?

Reply:
Changing this to moderation as moderator is a bit confusing on its purpose as its only 1 value